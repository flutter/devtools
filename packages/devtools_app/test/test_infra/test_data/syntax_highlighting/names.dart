// Copyright 2022 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

const $bool = null;
const bool$ = null;
const $num = null;
const num$ = null;
const $int = null;
const int$ = null;
const $double = null;
const double$ = null;
const $dynamic = null;
const dynamic$ = null;
const $true = null;
const true$ = null;
const $false = null;
const false$ = null;
const $null = null;
const null$ = null;
const $this = null;
const this$ = null;
const $super = null;
const super$ = null;
const $as = null;
const as$ = null;
const $try = null;
const try$ = null;
const $on = null;
const on$ = null;
const $catch = null;
const catch$ = null;
const $finally = null;
const finally$ = null;
const $throw = null;
const throw$ = null;
const $rethrow = null;
const rethrow$ = null;
const $break = null;
const break$ = null;
const $case = null;
const case$ = null;
const $continue = null;
const continue$ = null;
const $default = null;
const default$ = null;
const $do = null;
const do$ = null;
const $else = null;
const else$ = null;
const $for = null;
const for$ = null;
const $if = null;
const if$ = null;
const $in = null;
const in$ = null;
const $return = null;
const return$ = null;
const $switch = null;
const switch$ = null;
const $while = null;
const while$ = null;
const $sync = null;
const sync$ = null;
const $async = null;
const async$ = null;
const $await = null;
const await$ = null;
const $yield = null;
const yield$ = null;
const $assert = null;
const assert$ = null;
const $new = null;
const new$ = null;
const $abstract = null;
const abstract$ = null;
const $class = null;
const class$ = null;
const $enum = null;
const enum$ = null;
const $extends = null;
const extends$ = null;
const $extension = null;
const extension$ = null;
const $external = null;
const external$ = null;
const $factory = null;
const factory$ = null;
const $implements = null;
const implements$ = null;
const $get = null;
const get$ = null;
const $mixin = null;
const mixin$ = null;
const $native = null;
const native$ = null;
const $operator = null;
const operator$ = null;
const $set = null;
const set$ = null;
const $typedef = null;
const typedef$ = null;
const $with = null;
const with$ = null;
const $covariant = null;
const covariant$ = null;
const $static = null;
const static$ = null;
const $final = null;
const final$ = null;
const $const = null;
const const$ = null;
const $required = null;
const required$ = null;
const $late = null;
const late$ = null;
const $void = null;
const void$ = null;
const $var = null;
const var$ = null;
const $is = null;
const is$ = null;
