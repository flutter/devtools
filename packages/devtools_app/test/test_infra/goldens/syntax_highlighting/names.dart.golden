>// Copyright 2022 The Chromium Authors. All rights reserved.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// Use of this source code is governed by a BSD-style license that can be
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// found in the LICENSE file.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>
>const $bool = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const bool$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $num = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const num$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $int = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const int$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $double = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const double$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $dynamic = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const dynamic$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $true = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const true$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $false = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const false$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $null = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const null$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $this = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const this$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $super = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const super$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $as = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const as$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const $try = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const try$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $on = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const on$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const $catch = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const catch$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $finally = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const finally$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $throw = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const throw$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $rethrow = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const rethrow$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $break = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const break$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $case = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const case$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $continue = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const continue$ = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const $default = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const default$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $do = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const do$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const $else = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const else$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $for = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const for$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $if = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const if$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const $in = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const in$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const $return = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const return$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $switch = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const switch$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $while = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const while$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $sync = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const sync$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $async = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const async$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $await = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const await$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $yield = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const yield$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $assert = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const assert$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $new = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const new$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $abstract = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const abstract$ = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const $class = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const class$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $enum = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const enum$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $extends = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const extends$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $extension = null;
#^^^^^ storage.modifier.dart
#                 ^ keyword.operator.assignment.dart
#                   ^^^^ constant.language.dart
#                       ^ punctuation.terminator.dart
>const extension$ = null;
#^^^^^ storage.modifier.dart
#                 ^ keyword.operator.assignment.dart
#                   ^^^^ constant.language.dart
#                       ^ punctuation.terminator.dart
>const $external = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const external$ = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const $factory = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const factory$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $implements = null;
#^^^^^ storage.modifier.dart
#                  ^ keyword.operator.assignment.dart
#                    ^^^^ constant.language.dart
#                        ^ punctuation.terminator.dart
>const implements$ = null;
#^^^^^ storage.modifier.dart
#                  ^ keyword.operator.assignment.dart
#                    ^^^^ constant.language.dart
#                        ^ punctuation.terminator.dart
>const $get = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const get$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $mixin = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const mixin$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $native = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const native$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $operator = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const operator$ = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const $set = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const set$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $typedef = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const typedef$ = null;
#^^^^^ storage.modifier.dart
#               ^ keyword.operator.assignment.dart
#                 ^^^^ constant.language.dart
#                     ^ punctuation.terminator.dart
>const $with = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const with$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $covariant = null;
#^^^^^ storage.modifier.dart
#                 ^ keyword.operator.assignment.dart
#                   ^^^^ constant.language.dart
#                       ^ punctuation.terminator.dart
>const covariant$ = null;
#^^^^^ storage.modifier.dart
#                 ^ keyword.operator.assignment.dart
#                   ^^^^ constant.language.dart
#                       ^ punctuation.terminator.dart
>const $static = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const static$ = null;
#^^^^^ storage.modifier.dart
#              ^ keyword.operator.assignment.dart
#                ^^^^ constant.language.dart
#                    ^ punctuation.terminator.dart
>const $final = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const final$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $const = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const const$ = null;
#^^^^^ storage.modifier.dart
#             ^ keyword.operator.assignment.dart
#               ^^^^ constant.language.dart
#                   ^ punctuation.terminator.dart
>const $required = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const required$ = null;
#^^^^^ storage.modifier.dart
#                ^ keyword.operator.assignment.dart
#                  ^^^^ constant.language.dart
#                      ^ punctuation.terminator.dart
>const $late = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const late$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $void = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const void$ = null;
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^^^^ constant.language.dart
#                  ^ punctuation.terminator.dart
>const $var = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const var$ = null;
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^^^^ constant.language.dart
#                 ^ punctuation.terminator.dart
>const $is = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
>const is$ = null;
#^^^^^ storage.modifier.dart
#          ^ keyword.operator.assignment.dart
#            ^^^^ constant.language.dart
#                ^ punctuation.terminator.dart
